h2o-r/h2o-DESCRIPTION.template

```diff
@@ -55,6 +55,8 @@ Suggests: ggplot2,
           slam,
           bit64 (>= 0.9.7),
           data.table (>= 1.9.8),
+          rgl,
```

Comment:
I think it is better to be on a safe side here and specify the version that is being tested as the minimal version